sdk/src/test/scala/io/horizen/account/state/ContractInteropTestBase.scala

```diff
@@ -1,14 +1,22 @@
 package io.horizen.account.state
 
+import io.horizen.account.abi.ABIUtil.getABIMethodId
+import io.horizen.account.fork.ContractInteroperabilityFork
 import io.horizen.account.utils.BigIntegerUtil.toUint256Bytes
 import io.horizen.account.utils.{FeeUtils, Secp256k1}
 import io.horizen.evm._
-import io.horizen.utils.BytesUtils
+import io.horizen.fork.{ForkConfigurator, ForkManagerUtil, OptionalSidechainFork, SidechainForkConsensusEpoch}
+import io.horizen.utils.{BytesUtils, Pair}
 import org.junit.{After, Before}
 
 import java.math.BigInteger
+import java.util
+import scala.jdk.CollectionConverters.seqAsJavaListConverter
 
 abstract class ContractInteropTestBase extends MessageProcessorFixture {
+
+  // TODO the actual fork point is to be confirmed
```

Comment:
Actually for UTs we can use a value different from the 'official' one

Reply:
True. Removed the comment.